data/competencies.yml

```diff
@@ -407,6 +414,7 @@
   examples:
     - Can choose between similar Node libraries evaluating code quality, ease of integration, future maintenance, and security concerns
     - May be involved in evaluating paid-for third party supplier code
+    - Runs proof of concept demo to team or stakeholders
```

Comment:
I think this demonstrates a different skill to the competency, "Evaluates third-party software to use in projects". Maybe this should be an example in a different competency?